### DISCUSSION TITLE
Program Publish API  "On After Publish callback failed" error

### DISCUSSION THREAD

**Discussion by githubpiyush** [Sunbird-inQuiry/Community#46] [General Discussions]
We are trying to publish a program using API in VDN staging env and getting this error  :-

API :- **https://dockstaging.sunbirded.org/api/program/v1/publish**

`{
    "id": "api.v1.publish",
    "ver": "1.0",
    "ts": "2022-12-06T11:25:25.928Z",
    "params": {
        "resmsgid": "ae3eda80-7558-11ed-bcf5-df48f63ad78b",
        "msgid": null,
        "status": "failed",
        "err": "PRG_PRPUB"
    },
    "responseCode": "PRG_PRPUB02_ERR_PUBLISH_PROGRAM",
    "result": "On After Publish callback failed"
}`

Please let us know how can we resolve this error.

**Comment by Nadiya496** [comment]
@githubpiyush Please raise this on the cokreat community portal.
